resources/views/admin/layout_admin/layout.blade.php

```diff
@@ -9,8 +9,8 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css">
-    <link rel="stylesheet" href="{{url('../css')}}/style.css" type="text/css">
-    <link rel="stylesheet" href="{{url('../css')}}/all.css" type="text/css">
+     <link rel="stylesheet" href="/site/css/stylecss.css" type="text/css">
```

Comment:
sửa tên thực mục 

`layout_admin` => `layouts`

Reply:
Trong thư mục **layouts** tạo thêm thư mục **admin**

Trong thư mục admin tạo file `master.blade.php`

Move toàn bộ code của file `master.blade.php`

vào resources/views/admin/layout_admin/layout.blade.php